src/background.ts

```diff
@@ -0,0 +1,362 @@
+'use strict'
+import * as credentials from "./modules/credentials"
+import * as owaFetch from "./modules/owaFetch"
+
+// eslint-disable-next-line no-unused-vars
+const isFirefox = typeof browser !== 'undefined' && browser.runtime && browser.runtime.getBrowserInfo
```

Comment:
In this line the linter always errors because it can't find "browser". Installing the @types/firefox didn't fix it. Anyone know how to tell the linter it's ok?

Reply:
not precisely. But there are options for this.